src/main/java/org/concordion/api/CommandCall.java

```diff
@@ -90,4 +90,18 @@ public String getParameter(String camelCaseParameterName, String spinalCaseParam
         }
         return null;
     }
+
+    public boolean shouldExecuteWithNonExampleChildren() {

```

Comment:
Actually, went with a longer more descriptive name:

shouldExecuteEvenWhenAllChildCommandsAreExamples()
allChildCommandsAreExamples()

Then the test that uses it is:

        if (node.shouldExecuteEvenWhenAllChildCommandsAreExamples() || !node.allChildCommandsAreExamples()) {